ui/app/components/clients/date-range.ts

```diff
@@ -142,9 +138,11 @@ export default class ClientsDateRangeComponent extends Component<Args> {
   // used for CE date picker
   @action handleSave() {
     if (this.validationError) return;
+    // including the month param here as undefined allows the table month selection to reset when a new billing period is chosen
```

Comment:
It seems a little strange to do this here, since this is a query param it should probably be reset when the route refreshes